docs/api.md

```diff
@@ -951,6 +952,11 @@ Setting values for this object internally calls [bot.setControlState](#botsetcon
 
 ### Events
 
+#### "particle" (particle)
+
+Only emitted when a particle is created.
+ * `particle` - The particle created, type Json
```

Comment:
this gives no info

Reply:
then what should I write?

the paket shows ex.
```json
longDistance: false,
  x: -64,
  y: 68,
  offsetX: 8,
  offsetY: 8,
  offsetZ: 8,
  particleData: 16,
  particles: 9000,
  data: undefined
} { size: 50, name: 'world_particles', state: 'play' }
```

Reply:
this isnt a valid js object, what does the actual object look like?

Reply:
Exactly what I typed